common/src/main/java/io/netty/util/collection/PrimitiveCollections.java

```diff
@@ -122,8 +166,7 @@ public boolean containsValue(Object value) {
      *
      * @param <V> the value type stored in the map.
      */
-    private static final class UnmodifiableIntObjectMap<V> implements IntObjectMap<V>,
-            Iterable<IntObjectMap.Entry<V>> {
+    private static final class UnmodifiableIntObjectMap<V> implements IntObjectMap<V> {
```

Comment:
Hum not sure if we are using it...but it may be good for completeness to have this. We are using the empty map...and the unmodifiable should follow the same pattern...so shouldn't be too much extra work?